tests/src/autoload/functions.php

```diff
@@ -773,11 +773,11 @@ function loadDataset()
 
 /**
  * Test helper, search an item from its type and name
- *
- * @param string  $type
+ * @template T
+ * @param class-string<T>  $type
  * @param string  $name
  * @param boolean $onlyid
- * @return CommonDBTM|false the item, or its id
+ * @phpstan-return ($onlyid is true ? Int : T|false) Item of $type class, its id, false
```

Comment:
Since #18952, the `false` return value is no more possible.

```suggestion
 * @phpstan-return ($onlyid is true ? Int : T) Item of $type class, or its id
```